example/src/vcf2tiledb.cc

```diff
@@ -68,14 +102,51 @@ int main(int argc, char** argv)
     std::cerr << "Needs <loader_json_config_file>\n";
     exit(-1);
   }
+  auto loader_json_config_file = std::move(std::string(argv[optind]));
 #ifdef USE_GPERFTOOLS
```

Comment:
Only 3 tools have the gperftools functions for profiling - not sure whether it should be part of the core library itself (rather than the executables).